purge-docker-cluster.yml

```diff
@@ -1,43 +1,447 @@
 ---
 # This playbook purges a containerized Ceph cluster
-# It removes: packages, configuration files and ALL THE DATA
+# It removes: packages, containers, configuration files and ALL THE DATA
 
-- hosts:
-  - mons
-  - osds
+- name: confirm whether user really meant to purge the cluster
+
+  hosts:
+    - localhost
+
+  vars_prompt:
+    - name: ireallymeanit
+      prompt: Are you sure you want to purge the cluster?
+      default: 'no'
+      private: no
+
+  tasks:
+  - name: exit playbook, if user did not mean to purge cluster
+    fail:
+      msg: >
+        "Exiting purge-docker-cluster playbook, cluster was NOT purged.
+         To purge the cluster, either say 'yes' on the prompt or
+         or use `-e ireallymeanit=yes` on the command line when
+         invoking the playbook"
+    when: ireallymeanit != 'yes'
+
+
+- name: retrieve variables
+
+  hosts:
+    - all
+    - localhost
+
+  tasks:
+    - include_vars: roles/ceph-common/defaults/main.yml
+    - include_vars: roles/ceph-mon/defaults/main.yml
+    - include_vars: roles/ceph-osd/defaults/main.yml
+    - include_vars: roles/ceph-mds/defaults/main.yml
+    - include_vars: roles/ceph-rgw/defaults/main.yml
+    - include_vars: roles/ceph-nfs/defaults/main.yml
+    - include_vars: roles/ceph-restapi/defaults/main.yml
+    - include_vars: group_vars/all
+
+    - name: check if it is Atomic host
+      stat: path=/run/ostree-booted
+      register: stat_ostree
+
+    - name: set fact for using Atomic host
+      set_fact:
+        is_atomic: "{{ stat_ostree.stat.exists }}"
+
+
+- name: purge ceph mds cluster
+
+  hosts:
+    - mdss
```

Comment:
I will update this to use the respective variable. The reason I went this route is because I saw a similar thing being done in site.yml.sample and site-docker.yml.sample. It seems like we should update hosts in those playbooks to reference the respective `"{{ *_group_name }}"` variable as well.